### PR TITLE
[PCX-1369] Create automated deploy scripts to CocoaPods

### PR DESCRIPTION
**Pull request by Sorix**
**AS A** merchant developer
**I WANT** to access the latest iOS SDK version on CocoaPods as soon as it's been released
**SO THAT** I can use the latest and greatest version

### Acceptance criteria
1. CircleCI should automatically publish the binary to CocoaPods upon release
2. The CI should automatically increase version numbers as done in GoCD pipelines (look into major vs minor vs patch)

### CODE REVIEW

File: PayoneerCheckout.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1656,14 +1656,15 @@
...
-				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
+				INFOPLIST_FILE = Sources/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MARKETING_VERSION = 0.4;
```
Comment (siebeprojects) [inline PayoneerCheckout.xcodeproj/project.pbxproj:1667]: What is the MARKETING_VERSION used used for and why is it 0.4?

File: PayoneerCheckout.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1684,14 +1685,15 @@
...
```
Comment (siebeprojects) [inline PayoneerCheckout.xcodeproj/project.pbxproj:1696]: and here as well, 0.4

Comment (Sorix) [reply PayoneerCheckout.xcodeproj/project.pbxproj]: Removed that line

File: PayoneerCheckout.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Sorix) [reply PayoneerCheckout.xcodeproj/project.pbxproj]: Removed that line

**Review by siebeprojects — APPROVED** [review]

**Pull request merged**